keras_cv/layers/preprocessing/auto_contrast_test.py

```diff
@@ -90,3 +90,13 @@ def test_auto_contrast_properly_converts_value_range(self):
 
         self.assertTrue(tf.math.reduce_any(ys[0] == 0.0))
         self.assertTrue(tf.math.reduce_any(ys[0] == 1.0))
+
+    def test_augment_bbox_dict_input(self):
```

Comment:
Thanks for including all of these unit tests!

Can we create a single `with_bounding_boxes_test.py` file based on: https://github.com/keras-team/keras-cv/blob/master/keras_cv/layers/preprocessing/with_labels_test.py to handle all of these that do not interact with bounding_boxes?